Logitech B500 calibration file is empty

Results in this horrific error:

```
[0:00:00.000045] === LOG STARTED ===
[0:00:03.278024] sr.robot3.robot INFO - sr.robot3 version 2022.0.0a2
[0:00:03.522923] [ WARN:0] global /usr/src/debug/opencv/4.5.2-r0/git/modules/videoio/src/cap_gstreamer.cpp (597) isPipelinePlaying OpenCV | GStreamer warning: GStreamer: pipeline have not been created
[0:00:04.330512] sr.robot3.robot INFO - Found Student Robotics v4 Motor Board - SR0KK14
[0:00:04.339270] sr.robot3.robot INFO - Found Student Robotics v4 Power Board - XXXXXXXXXXXXXXX
[0:00:04.342310] sr.robot3.robot INFO - Found Student Robotics v4 Motor Board - SR0RJ1R
[0:00:04.348336] sr.robot3.robot INFO - Found Zoloto Camera Board - ZOLOTOCAM
[0:00:04.351408] sr.robot3.robot INFO - Waiting for start signal
[0:00:08.686068] sr.robot3.robot INFO - Start signal received; continuing.
[0:00:08.699182] cv2.error: OpenCV(4.5.2) /usr/src/debug/opencv/4.5.2-r0/git/modules/core/src/persistence.cpp:706: error: (-215:Assertion failed) buf in function 'open'
[0:00:08.700646] 
[0:00:08.702086] 
[0:00:08.703491] The above exception was the direct cause of the following exception:
[0:00:08.704776] 
[0:00:08.706135] Traceback (most recent call last):
[0:00:08.707491]   File "/tmp/astprocd-qovkevz7/robot.py", line 5, in <module>
[0:00:08.709767]     r.camera.see()
[0:00:08.711419]   File "/usr/lib/python3.9/site-packages/j5_zoloto/board.py", line 54, in see
[0:00:08.713089]     return self._camera.see(eager=eager)
[0:00:08.714838]   File "/usr/lib/python3.9/site-packages/j5_zoloto/component.py", line 91, in see
[0:00:08.716477]     return list(self._backend.process_frame_eager(self._identifier))
[0:00:08.718120]   File "/usr/lib/python3.9/site-packages/zoloto/cameras/base.py", line 122, in process_frame_eager
[0:00:08.719738]     calibration_params = self.get_calibrations()
[0:00:08.721332]   File "/usr/lib/python3.9/site-packages/zoloto/cameras/base.py", line 34, in get_calibrations
[0:00:08.722971]     return parse_calibration_file(self.calibration_file)
[0:00:08.724580]   File "/usr/lib/python3.9/site-packages/zoloto/calibration.py", line 29, in parse_calibration_file
[0:00:08.726235]     storage = FileStorage(str(calibration_file), FILE_STORAGE_READ)
[0:00:08.727793] SystemError: <class 'cv2.FileStorage'> returned a result with an error set
```

--- a/sr/robot3/vision/calibrations/Logitech B500.xml
+++ b/sr/robot3/vision/calibrations/Logitech B500.xml
@@ -1,0 +1,36 @@
+<?xml version="1.0"?>
+<opencv_storage>
+<calibrationDate>"Thu 11 Nov 2021 17:34:35 GMT"</calibrationDate>
+<framesCount>29</framesCount>
+<cameraResolution>
+  1280 1024</cameraResolution>
+<cameraMatrix type_id="opencv-matrix">
+  <rows>3</rows>
+  <cols>3</cols>
+  <dt>d</dt>
+  <data>
+    9.3012007264063072e+02 0. 7.0226810811512655e+02 0.
+    9.3012007264063072e+02 4.7705847029705535e+02 0. 0. 1.</data></cameraMatrix>
+<cameraMatrix_std_dev type_id="opencv-matrix">
+  <rows>4</rows>
+  <cols>1</cols>
+  <dt>d</dt>
+  <data>
+    0. 1.4841088432691212e+01 4.7206079728271524e+00
+    4.2203301535161479e+00</data></cameraMatrix_std_dev>
+<dist_coeffs type_id="opencv-matrix">
+  <rows>1</rows>
+  <cols>5</cols>
+  <dt>d</dt>
+  <data>
+    -9.2701468925095482e-02 2.9255285923072902e-02
+    -7.9310903187861364e-03 1.1943725785898814e-02 0.</data></dist_coeffs>
+<dist_coeffs_std_dev type_id="opencv-matrix">
+  <rows>5</rows>
+  <cols>1</cols>
+  <dt>d</dt>
+  <data>
+    4.4573481761620787e-03 4.5353543208646336e-03 9.4635517516124560e-04
+    8.7616716255079267e-04 0.</data></dist_coeffs_std_dev>
+<avg_reprojection_error>7.7260806740601118e-01</avg_reprojection_error>
+</opencv_storage>

